### PR TITLE
Complete Lab

### PR DESCRIPTION
**Pull request by Tomm1128**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+// First function should be a regular function to divide two numbers.
+
+function divide (num1, num2){
+        return num1 / num2;
+}
```
Comment (stephenmckeon) [inline index.js:5]: This can also be an arrow function 😺 
```suggestion
const divide = (num1, num2) => num1 / num2
```

**Review by stephenmckeon — COMMENTED** [review]
Nice work!

File: index.js (hunk 1 above)
Comment (Tomm1128) [reply index.js:5]: Yeah, the tests seem to want to first function to be a regular function. It was confusing. Like I originally wrote it as an arrow function but rewrote it when reading the tests.

Comment (stephenmckeon) [reply index.js:5]: Gotchya, yeah makes sense!

**Pull request closed without merging**